supporting_content/use_cases/compare_billed_cost_per_subaccount_to_budget.md

```diff
@@ -0,0 +1,54 @@
+# Use Case
+
+Compare billed cost per subaccount to budget
+
+## Use Case ID
+14383
+
+## Supported Specification Version
+1.0
+
+## Description
+Product wants to confirm charges are in line with expectations (budgets) for a business which has all its cost within a single subaccount. The ChargePeriod is used for filtering to capture items that apply to the period only (excludes adjustments from previous periods that were applied in the current BillingPeriod).
+
+## 1st Degree Columns
+* Provider
+* Sub Account ID
+* Sub Account Name
+* Billed Cost
+* Charge Category
+* Charge Period Start
+* Charge Period End
+
+## 2nd Degree Columns
+
+
+
+## Example SQL Query
+```
+SELECT
+  ProviderName,
+  SubAccountId,
+  SubAccountName,
+  SUM(BilledCost) AS TotalBilledCost
+FROM focus_data_table
+WHERE ChargeCategory = 'Usage'
+  AND ChargePeriodStart >= ? and ChargePeriodEnd <= ?
```

Comment:
Why are we only showing usage? That doesn't seem like it's something that can be compared to a budget since purchases, adjustments, credits, and possibly taxes would impact budgets.
```suggestion
WHERE ChargePeriodStart >= ? AND ChargePeriodEnd <= ?
```